mesonbuild/compilers/c.py

```diff
@@ -17,6 +17,7 @@
 from .. import mlog
 from .. import coredata
 from ..mesonlib import EnvironmentException, version_compare, Popen_safe, listify
+from ..mesonlib import MesonException
```

Comment:
Merge with previous line?

Reply:
Sure, actually upon rebasing there are two lines anyway, but this fits on the second one nicely.